tutorials/bert_e2e.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import shutil
+import re
+import tempfile
+import functools
+from pathlib import Path
+
+from onnxruntime_extensions.tools import add_pre_post_processing_to_model as add_ppp
+import onnxruntime_extensions
+
+# for tokenizer
+import transformers
+import numpy as np
+import onnxruntime
+
+
+# avoid loading model from hugging-face multiple times, it's time consuming
+@functools.lru_cache
```

Comment:
Does huggingface not cache locally itself like torchvision does?

Reply:
I would say no. It gives me a new model each time.